Packs/EmailCommunication/ReleaseNotes/2_0_46.md

```diff
@@ -0,0 +1,6 @@
+
+#### Scripts
+
+##### SendEmailReply
+
+Fixed an issue where the script would fail in the **XSOAR - Email Communication Signature** list could not be loaded.
```

Comment:
```suggestion
Fixed an issue where the script would fail in the **XSOAR - Email Communication Signature** if a list could not be loaded.
```

Reply:
Thank you, corrected:)

I actually intended to write:
"""
Fixed an issue where the script would fail _**if**_ the **XSOAR - Email Communication Signature** list could not be loaded.
"""